packages/cxl-ui/src/components/cxl-jw-player/index.html.js

```diff
@@ -0,0 +1,60 @@
+import { html, nothing } from 'lit';
+import '@vaadin/icon';
+import '@vaadin/text-field';
+
+// eslint-disable-next-line func-names
+export const template = function () {
+  return html`
+    ${this.ready
+      ? html`
+          <div class="grid height-100" id="container">
+            <slot></slot>
+            ${this.captions
+              ? html`
+                  <div class="center search padding">
+                    <vaadin-text-field
+                      @input=${this._debouncedSearch}
+                      clear-button-visible
+                      id="search"
+                      placeholder="Type and press enter to search"
+                      theme="small"
+                    >
+                      <vaadin-icon slot="prefix" icon="lumo:search"></vaadin-icon>
+                    </vaadin-text-field>
+                    <div id="searchResultCount">
```

Comment:
We don't use `camelCase` for HTML classes and IDs

```suggestion
                    <div id="search-result-count">
```

Reply:
This should be fixed.